test/test_service.rb

```diff
@@ -109,4 +109,28 @@ def test_get_service
                      'http://localhost:8080/api/v1beta3/namespaces/development/services/redis-slave',
                      times: 1)
   end
+
+  def test_update_service
```

Comment:
1. what field is this test updating ?
2. this example will not work on a real k8s server as there are many required fields that are not specified. (for example - resourceVersion)